services/search/tests/routes/test_filter.py

```diff
@@ -81,7 +81,7 @@ def test_execute_filter_query(index_file_location: str, columns: Union[list[str]
     assert num_rows_total == 2
     expected = pa.Table.from_pydict({"__hf_index_id": [3], "name": ["Simone"], "gender": ["female"], "age": [30]})
     if isinstance(columns, list):
-        for column in set(expected.column_names) - set([ROW_IDX_COLUMN] + columns):
+        for column in set(expected.column_names) - set(columns):
             expected = expected.drop(column)
     assert pa_table == expected
```

Comment:
I think the expected Table should have `__hf_index_id ` otherwise we can't format the filter output to have the right row_idx